client/src/modules/shopping-list/model/reducer.js

```diff
@@ -33,7 +38,18 @@ const items = (state = itemsInitialState, action) => {
   }
 };
 
-const uiStatus = (state = initialStatus, action) => {
+export const shoppingLists = (state = [], action) => {
```

Comment:
shouldn't we have two separate reducers for items (products) and shopping lists?

Reply:
yes we have separated reducers for items/products and shopping lists

Reply:
Sorry, I meant that in in separate files